cekit/cli.py

```diff
@@ -178,6 +178,10 @@ def run(self):
 
             if self.args.redhat:
                 tools.cfg['common']['redhat'] = True
+
+            if tools.cfg['common']['redhat'] and 'cache_url' not in tools.cfg['common']:
+                tools.cfg['common']['cache_url'] = \
+                    'http://ce-cacher.usersys.redhat.com/get?#algorithm#=#hash#'
```

Comment:
Internal URL in upstream code?

Reply:
seems like its common practice https://pagure.io/odcs/blob/master/f/client/contrib/odcs#_24

Reply:
I'm not saying no, just this would be precedence in our code since we haven't done anything like this so far. This, of course, makes things much easier for internal users.

Reply:
ok lets ask @jmtd and @rcernich for opinions?

Reply:
Agree.

Reply:
This is only default. You can still override it by defining in ~/.cekit/config as you can do now..

Reply:
And anybody running cekit who doesn't update their default will be trying to hit this url.

Reply:
Yep you are probably rigth. Closing this. I'm still thinking more and more of moving this redhat related stuff into cekit-redhat-plugin rpm or something more feasible;